### DISCUSSION TITLE
D'cent app. and Walletconnect

### DISCUSSION THREAD

**Discussion by rdamerch** [WalletConnect/walletconnect-monorepo#2946] [Developer Support]
I needed to synchronize my defi wallet (D'cent app.) manually through the project mainnet server to establish a secure dApps connection and allow my wallet connect successfully using the wallet connect app. After several attempts the project mainnet server locked my XLM tokens from my wallet in to mainnet server. any help will be appreciated please advise thank you

**Comment by chadyj** [comment]
What dapp are you trying to connect to?

**Reply by rdamerch** [reply]
I used D'cent app. and Bifrost app. via wallet-connect. Thank you

**Comment by rdamerch** [comment]
D'cent wallet app. and Bifrost wallet app. via walletconnect. please advise thank you

________________________________
